packages/algob/src/lib/deployer.ts

```diff
@@ -32,6 +35,7 @@ export class AlgobDeployerImpl implements AlgobDeployer {
     this.cpData = cpData;
     this.loadedAsaDefs = asaDefs;
     this.algoOp = algoOp;
+    // this.compileOp = new CompileOp(this.algoOp.algodClient);
```

Comment:
If you see on line 37 I already pass `algoOp` that is created before the `Deployer`. This means that Deployer doesn't need to create `CompileOp` itself.

Reply:
yes